.env.sample

```
@@ -7,7 +7,7 @@ DRYRUN = "True"
 DEBUG_LEVEL = "INFO"
 
 ## If set to true then the script will only run once and then exit
-RUN_ONLY_ONCE = "False"
+RUN_ONLY_ONCE = "True"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)